cosmos/config.py

```diff
@@ -212,6 +212,8 @@ class ProfileConfig:
     :param target_name: The name of the dbt target to use.
     :param profiles_yml_filepath: The path to a profiles.yml file to use.
     :param profile_mapping: A mapping of Airflow connections to dbt profiles.
+    :param dbt_config_vars: Dictionary of dbt configs for the project. This argument overrides configs defined in your profiles.yml
+        file. The dictionary is dumped to a yaml string. Details https://docs.getdbt.com/docs/core/connect-data-platform/profiles.yml
```

Comment:
The arg description here says it overrides configs in the profiles.yml, but if a user uses `profiles_yml_filepath` instead of `profile_mapping` it does not update the profile config in that file. It should be made clear that configs are added to the mapped profile file and an exception should be raised if `profiles_yml_filepath` is used with `dbt_config_vars`.